src/controllers/orderController.js

```diff
@@ -92,6 +128,17 @@ const deleteOrder = async (req, res) => {
   await Order.findByIdAndDelete(orderId); // or more simply can just call delete on the documnent we've already fetched:  await order.remove()
 
   res.status(StatusCodes.OK).json({ msg: 'Success! Order was deleted' });
+  /*
+  #swagger.summary = 'Delete an order by id'
+  #swagger.description = '**ROLE REQUIRED:** user'
+  #swagger.parameters['id'] = {
+    description: 'Mongo ObjectID of the order to delete',
+  }
+  #swagger.responses[200] = {
+    description: 'The order was successfully deleted.',
```

Comment:
We can also include an example of what the response body would look like:

```suggestion
    description: 'The order was successfully deleted.',
    schema: { msg: 'Success! Order was deleted' }
```